src/main/scala/com/microsoft/hyperspace/util/LogicalPlanUtils.scala

```diff
@@ -35,4 +42,35 @@ object LogicalPlanUtils {
       case _ => false
     }
   }
+
+  def getAllAvailableIndexes(states: Seq[String] = Seq()): Seq[IndexLogEntry] = {
```

Comment:
Let's not expose this API since it has a tight dependency on spark session, context, etc.